posts/post-router.js

```diff
@@ -0,0 +1,72 @@
+const express = require("express");
+const router = express.Router();
+
+const dbp = require("./postDb");
+const dbu = require("../users/userDb");
+
+//middleware
+
+//Routes
+
+router.get("/", async (req, res) => {
+  try {
+    const posts = await req.body;
+    dbp.get().then(posts => {
+      res.status(200).json({ posts: posts });
```

Comment:
```suggestion
      res.status(200).json({ posts });
```
Unnecessary to have both the key/value if they're the same name 😄